### PR TITLE
Fix Swift SDK bundles not working due to quarantine

### PR DESCRIPTION
**Pull request by MaxDesiatov**
When using `swift experimental-sdk install` subcommand with bundle archives or bundles unpacked from archives downloaded manually in the browser, installed executables are quarantined and trigger an error message when used by `swift build`

Steps To Reproduce:
1. Download a destination bundle archive
2. Unpack and install the bundle with `swift experimental-destination install`
3. Confirm that the destination is installed and note its ID with `swift experimental-destination list`
4. Build using the newly installed destination with `swift build --experimental-destination-selector <destination-id>`

Actual result:
Building with this destination triggers an error: "`swift-driver` cannot be opened because the developer cannot be verified."

rdar://107392863

### Modifications:

We now suggest that users run `xattr -r -d -s com.apple.quarantine` on the bundles they trust to remove the quarantine attribute.

### Result

The error message is clear and actionable.

### CODE REVIEW

File: Sources/PackageModel/SwiftSDKBundle.swift [hunk 1]
```diff
@@ -294,6 +295,20 @@ public struct SwiftSDKBundle {
         }
 
         try fileSystem.copy(from: unpackedBundlePath, to: installedBundlePath)
+
+        #if os(macOS)
+        // Remove the quarantine attribute from bundles downloaded manually in the browser.
+        let result = try Process.popen(
+            arguments: ["xattr", "-d", "-r", "-s", "com.apple.quarantine", installedBundlePath.pathString]
```
Comment (neonichu) [inline Sources/PackageModel/SwiftSDKBundle.swift:302]: Could we use `removexattr` (or similar) instead of shelling out?

**Review by tomerd — APPROVED** [review]

File: Sources/PackageModel/SwiftSDKBundle.swift (hunk 1 above)
Comment (tomerd) [reply Sources/PackageModel/SwiftSDKBundle.swift:302]: +1 if possible

**Comment by MaxDesiatov** [comment]
https://github.com/apple/swift-tools-support-core/pull/414

@swift-ci smoke test

**Review by tomerd — APPROVED** [review]

File: Sources/PackageModel/SwiftSDKBundle.swift [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 import Basics
 
+import protocol TSCBasic.FileSystem
```
Comment (tomerd) [inline Sources/PackageModel/SwiftSDKBundle.swift:15]: nit: sort imports

**Comment by MaxDesiatov** [comment]
@swift-ci smoke test

**Comment by MaxDesiatov** [comment]
@swift-ci test windows

**Comment by MaxDesiatov** [comment]
@swift-ci test windows

**Comment by MaxDesiatov** [comment]
@swift-ci smoke test

**Comment by MaxDesiatov** [comment]
@swift-ci test windows

**Comment by MaxDesiatov** [comment]
@swift-ci test windows

**Pull request merged**